common/src/main/java/li/cil/scannable/common/config/CommonConfig.java

```diff
@@ -118,4 +122,19 @@ public final class CommonConfig {
     private static Set<ResourceLocation> getDefaultCommonOreTags() {
         throw new AssertionError();
     }
+
+    @Path("chests") @WorldRestart
+    @Comment("Registry names of blocks considered 'chests', requiring the chest scanner module.")
+    @ItemType(ResourceLocation.class)
+    public static Set<ResourceLocation> commonChestBlocks = new HashSet<>();
+
+    @ExpectPlatform
+    private static Set<ResourceLocation> getDefaultChestsTags() {
+        throw new AssertionError();
+    }
+
+    @Path("chests") @WorldRestart
+    @Comment("Registry names of blocks considered 'chests', requiring the chest scanner module.")
+    @ItemType(ResourceLocation.class)
+    public static Set<ResourceLocation> commonChestTags = getDefaultChestsTags();
```

Comment:
Please stick to the order of things in this class, i.e. fields first, then methods.
So have the new fields above the `getDefaultCommonOreTags()`.